public/js/pimcore/startup.js

```diff
@@ -971,5 +971,7 @@ pimcore.helpers.unload = function () {
 };
 
 L.Icon.Default.imagePath = '../bundles/pimcoreadmin/build/admin/images/';
-pimcore.wysiwyg = {};
-pimcore.wysiwyg.editors = [];
+if (!pimcore.wysiwyg) {
```

Comment:
doesn't look the right fix, it is just disabling this block this https://github.com/pimcore/admin-ui-classic-bundle/blob/1976b8f869ad450e1b3d2b9e4aebcc47f3487e7e/public/js/pimcore/object/tags/wysiwyg.js#L182-L186

Reply:
re-assigning to @robertSt7 as he is more familiar with these changes

Reply:
without this, the editor in objects does not see the wysiwyg setting

Reply:
without this, the editor in objects does not see the wysiwyg setting

Reply:
ah ok, i see what you mean, then it's valid